agent/src/main/java/io/pyroscope/javaagent/JFRProfilerDelegate.java

```diff
@@ -46,9 +54,9 @@ public synchronized void start() {
             commands.add("JFR.start");
             commands.add("name=" + RECORDING_NAME);
             commands.add("filename=" + tempJFRFile.getAbsolutePath());
-            commands.add("settings=pyroscope");
+            commands.add("settings=" + jfrSettingsPath);
             ProcessBuilder processBuilder = new ProcessBuilder(commands);
-            Process process = processBuilder.start();
+            Process process = processBuilder.inheritIO().start();
```

Comment:
am i understanding correctly that this will redirect the commands stdout to the java process stdout?

If so, I suggest we read it manualy and log it with our logger interface so users can disable it